Source/UltraleapTrackingCore/UltraleapTracking.Build.cs

```diff
@@ -171,36 +182,42 @@ public bool LoadLeapLib(ReadOnlyTargetRules Target)
 
 				string PlatformString = Target.Platform.ToString();
 
-				//Lib
-				PublicAdditionalLibraries.Add(Path.Combine(LibraryPath, PlatformString, "LeapC.lib"));
-
-				//System.Console.WriteLine("plugin using lib at " + Path.Combine(LibraryPath, PlatformString, "LeapC.lib"));
-
-				if (IsEnginePlugin())
+				string ThirdPartyDllPath = Path.Combine(LibraryPath, PlatformString, "LeapC.dll");
+				string ThirdPartyDllManifPath = Path.Combine(LibraryPath, PlatformString, "LeapC.dll.manifest");
+                string BinDLLPath = Path.Combine(BinariesPath, PlatformString, "LeapC.dll");
+				string BinDLLManifPath = Path.Combine(BinariesPath, PlatformString, "LeapC.dll.manifest");
+                //Lib
+                PublicAdditionalLibraries.Add(Path.Combine(LibraryPath, PlatformString, "LeapC.lib"));
+                //System.Console.WriteLine("plugin using lib at " + Path.Combine(LibraryPath, PlatformString, "LeapC.lib"));
+                // Copy third party DLLs to the BinariesPath 
+                RuntimeDependencies.Add(BinDLLPath, ThirdPartyDllPath);
+                RuntimeDependencies.Add(BinDLLManifPath, ThirdPartyDllManifPath);
+
+                // This will copy dlls if not copied already
+                CopyToBinaries(ThirdPartyDllPath);
+                CopyToBinaries(ThirdPartyDllManifPath);
+
+                if (IsEnginePlugin())
 				{
 					PublicDelayLoadDLLs.Add("LeapC.dll");
-					RuntimeDependencies.Add(Path.Combine(BinariesPath, PlatformString, "LeapC.dll"));
-				}
+                }
 				//Engine plugin, just add the dependency path
 				else
 				{
 					//DLL
-					string PluginDLLPath = Path.Combine(BinariesPath, PlatformString, "LeapC.dll");
-				
-					System.Console.WriteLine("Project plugin detected, using dll at " + PluginDLLPath);
+					System.Console.WriteLine("Project plugin detected, using dll at " + BinDLLPath);
 
-					RuntimeDependencies.Add(PluginDLLPath);
-					if (!Target.bBuildEditor)
+                    if (!Target.bBuildEditor)
 					{
 						PublicDelayLoadDLLs.Add("LeapC.dll");
 					}
 				}
 			}
 			else if (Target.Platform == UnrealTargetPlatform.Mac)
 			{
-				IsLibrarySupported = true;	
-                Setlib("Mac", "libLeapC.5.dylib");
-                Setlib("Mac", "libLeapC.5_intel.dylib");
+				//IsLibrarySupported = true;	
+                //Setlib("Mac", "libLeapC.5.dylib");
+                //Setlib("Mac", "libLeapC.5_intel.dylib");
```

Comment:
Should this still be commented out?

Reply:
yea, for the marketplace plugin will need to disable MAC support